android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -84,6 +86,27 @@ public void prepare(final String fileName, final Integer key, final ReadableMap
     
     player.setAudioStreamType(audioStreamTypeFinal);
 
+    if (module.category != null) {
```

Comment:
@ohtangza Oh! Setting `category` is supported here!

Reply:
Oh, that's true.